automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -88,7 +88,7 @@ def setup_products_for_upgrade(product, os_version):
     return sat_host, cap_hosts, clients6, clients7
 
 
-def product_upgrade(product):
```

Comment:
Set both clients6 and clients7 to None by default.  So that wnevever it is set to None, the upgrade will consider this as to run with rhevm sat and will generate clients for rhel6 and rhel7 on docker.